code/_globalvars/bitfields.dm

```diff
@@ -582,3 +582,12 @@ DEFINE_BITFIELD(bot_mode_flags, list(
 	"SAPIENCE_ALLOWED" = BOT_MODE_CAN_BE_SAPIENT,
 	"STARTS_POSSESSABLE" = BOT_MODE_ROUNDSTART_POSSESSION,
 ))
+
+DEFINE_BITFIELD(upgrade, list(
```

Comment:
Name this list to something more meaningful like rcd_upgrades_list. Also make use of this bitfield in the VV var edit menu for toggling different upgrade flags

Reply:
The name has to match the name of the var on /obj/item/construction, and yeah these bitfield definitions are automatically used by the VV menu when you edit bitflag vars

Reply:
> The name has to match the name of the var on /obj/item/construction


These upgrades only apply to  /obj/item/construction/rcd. Stuff like RPLD, RTD & RLD (subtypes of /obj/item/construction) don't benifit from anti desurption and many of the other upgrades.

Move the bitfield to /obj/item/construction/rcd

Reply:
Also move the examine code down to /obj/item/construction/rcd from the above comment